scripts/gold_accounting.py

```diff
@@ -207,12 +207,12 @@ def get_transactions(cls, start_date=None, end_date=None):
             query = query.filter(cls.date >= start_date)
         if end_date:
             query = query.filter(cls.date < end_date)
-        
+
         query = query.order_by(cls.date, cls.trans_id)
-        transactions = query.all()
+        query = query.yield_per(1000)   # retrieve 1000 rows at a time
```

Comment:
Eh I like it.  It made me originally think he'd only be getting the _first_ 1000, but the comment makes it clear that it will just iterate through it all and in batches.